include/tvm/runtime/packed_func.h

```diff
@@ -394,8 +409,9 @@ class TVMRetValue : public TVMPODValue_ {
   using TVMPODValue_::operator int;
   using TVMPODValue_::operator bool;
   using TVMPODValue_::operator void*;
-  using TVMPODValue_::operator TVMArray*;
+  using TVMPODValue_::operator DLTensor*;
```

Comment:
Why do we pass the underlying pointer instead of the container?

Reply:
We store the by pointer. But from user's perspective, they pass in containers

Reply:
I see now, looking at Python code more carefully.